.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "2.7"
+before_install:
+  - git clone https://github.com/UAVCAN/libuavcan
+  - pip install uavcan
+script:
+  - libuavcan/libuavcan/dsdl_compiler/libuavcan_dsdlc -v uavcan/
```

Comment:
Perhaps we should avoid dependency on libuavcan and use PyUAVCAN directly? Here's the tutorial: http://uavcan.org/Implementations/Pyuavcan/Tutorials/4._Parsing_DSDL_definitions/. The necessary staements can be executed directly from the command line arguments. For example (I didn't test this):

```bash
python3 -c "from pyuavcan import dsdl; dsdl.parse_namespaces('uavcan')"
```